.github/workflows/cd.yaml

```
@@ -0,0 +1,29 @@
+        uses: actions/checkout@v4
+
+      - name: Build & Deploy
+        uses: appleboy/ssh-action@v1
+        with:
+          host: ${{ secrets.SERVER_IP }}
+          username: ${{ secrets.USER_NAME }}
+          key: ${{ secrets.PRIVATE_KEY }}
+          port: ${{ secrets.SSH_PORT }}
+          timeout: 40s
+          script: |
+            cd ~/HoYo.gg 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)